pkg/kube/activity.go

```diff
@@ -756,6 +761,10 @@ func (k *PromoteStepActivityKey) OnPromoteUpdate(jxClient versioned.Interface, n
 	if k.ApplicationURL != "" {
 		ps.ApplicationURL = k.ApplicationURL
 	}
+
+	if ok, _ := IsTektonEnabled(kubeClient, ns); ok && p.Status != v1.ActivityStatusTypeRunning {
+		a.Spec.Status = p.Status
```

Comment:
Newbie question, are you sure this is enough ? 
According to https://github.com/jenkins-x/jx/issues/5755, there is also `workflowStatus: Running`

Reply:
`workflowStatus` is only used at all in static masters, so no need to worry about it here.